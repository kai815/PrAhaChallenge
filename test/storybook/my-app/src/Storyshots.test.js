import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';
import { createSerializer } from '@emotion/jest';
initStoryshots({
  integrityOptions: { cwd: __dirname }, // it will start searching from the current directory
  test: multiSnapshotWithOptions(),
  snapshotSerializers: [createSerializer()],
});
