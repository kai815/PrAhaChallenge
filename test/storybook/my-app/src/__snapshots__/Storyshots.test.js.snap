// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Example/Board Display All O 1`] = `
<div>
  <div
    className="css-1c7d9u3-Board"
  >
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      O
    </button>
  </div>
  <div
    className="css-1c7d9u3-Board"
  >
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      O
    </button>
  </div>
  <div
    className="css-1c7d9u3-Board"
  >
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      O
    </button>
  </div>
</div>
`;

exports[`Storyshots Example/Board Display All Triangle 1`] = `
<div>
  <div
    className="css-1c7d9u3-Board"
  >
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
  <div
    className="css-1c7d9u3-Board"
  >
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
  <div
    className="css-1c7d9u3-Board"
  >
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
</div>
`;

exports[`Storyshots Example/Board Display All X 1`] = `
<div>
  <div
    className="css-1c7d9u3-Board"
  >
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      X
    </button>
  </div>
  <div
    className="css-1c7d9u3-Board"
  >
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      X
    </button>
  </div>
  <div
    className="css-1c7d9u3-Board"
  >
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="css-uqotkq-Square"
      onClick={[Function]}
    >
      X
    </button>
  </div>
</div>
`;

exports[`Storyshots Example/Game Game Initial 1`] = `
<div
  className="css-17xd72k-Game"
>
  <div
    className="css-vt0g4w-Game"
  >
    <div>
      <div
        className="css-1c7d9u3-Board"
      >
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          
        </button>
      </div>
      <div
        className="css-1c7d9u3-Board"
      >
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          
        </button>
      </div>
      <div
        className="css-1c7d9u3-Board"
      >
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          
        </button>
      </div>
    </div>
  </div>
  <div
    className="css-vt0g4w-Game"
  >
    <div>
      Next Player X's turn
    </div>
    <ol>
      <li>
        <button
          onClick={[Function]}
        >
          Go to game start
        </button>
      </li>
    </ol>
  </div>
</div>
`;

exports[`Storyshots Example/Game Gmae All Triangle 1`] = `
<div
  className="css-17xd72k-Game"
>
  <div
    className="css-vt0g4w-Game"
  >
    <div>
      <div
        className="css-1c7d9u3-Board"
      >
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          △
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          △
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          △
        </button>
      </div>
      <div
        className="css-1c7d9u3-Board"
      >
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          △
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          △
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          △
        </button>
      </div>
      <div
        className="css-1c7d9u3-Board"
      >
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          △
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          △
        </button>
        <button
          className="css-uqotkq-Square"
          onClick={[Function]}
        >
          △
        </button>
      </div>
    </div>
  </div>
  <div
    className="css-vt0g4w-Game"
  >
    <div>
      Winner: △
    </div>
    <ol>
      <li>
        <button
          onClick={[Function]}
        >
          Go to game start
        </button>
      </li>
    </ol>
  </div>
</div>
`;

exports[`Storyshots Example/Move Desc 4 1`] = `"Go to move #4"`;

exports[`Storyshots Example/Move Desc Initial 1`] = `"Go to game start"`;

exports[`Storyshots Example/Move Move History 3 1`] = `
Array [
  <li>
    <button
      onClick={[Function]}
    >
      Go to game start
    </button>
  </li>,
  <li>
    <button
      onClick={[Function]}
    >
      Go to move #1
    </button>
  </li>,
  <li>
    <button
      onClick={[Function]}
    >
      Go to move #2
    </button>
  </li>,
]
`;

exports[`Storyshots Example/Move Move History Initial 1`] = `
<li>
  <button
    onClick={[Function]}
  >
    Go to game start
  </button>
</li>
`;

exports[`Storyshots Example/Sqaure Display O 1`] = `
<button
  className="css-uqotkq-Square"
  onClick={[Function]}
>
  O
</button>
`;

exports[`Storyshots Example/Sqaure Display Triangle 1`] = `
<button
  className="css-uqotkq-Square"
  onClick={[Function]}
>
  △
</button>
`;

exports[`Storyshots Example/Sqaure Display X 1`] = `
<button
  className="css-uqotkq-Square"
  onClick={[Function]}
>
  X
</button>
`;

exports[`Storyshots Example/Status Winner Null Next X 1`] = `"Next Player X's turn"`;

exports[`Storyshots Example/Status Winner X 1`] = `"Winner: X"`;

exports[`Storyshots Example/Status Winner Y 1`] = `"Winner: Y"`;
